test/gtest/glu_bwd.cpp

```diff
@@ -0,0 +1,106 @@
+/*******************************************************************************
+ *
+ * MIT License
+ *
+ * Copyright (c) 2024 Advanced Micro Devices, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ *******************************************************************************/
+
+#include "glu.hpp"
+#include <miopen/env.hpp>
+
+using float16 = half_float::half;
+
+MIOPEN_DECLARE_ENV_VAR_STR(MIOPEN_TEST_FLOAT_ARG)
+MIOPEN_DECLARE_ENV_VAR_BOOL(MIOPEN_TEST_ALL)
+
+namespace glu {
+
+std::string GetFloatArg()
+{
+    const auto& tmp = env::value(MIOPEN_TEST_FLOAT_ARG);
+    if(tmp.empty())
+    {
+        return "";
+    }
+    return tmp;
+}
+
+struct GPU_GLU_bwd_FP32 : GLUBwdTest<float>
+{
+};
+
+struct GPU_GLU_bwd_FP16 : GLUBwdTest<float16>
+{
+};
+
+struct GPU_GLU_bwd_BFP16 : GLUBwdTest<bfloat16>
+{
+};
+
+} // namespace glu
+using namespace glu;
```

Comment:
```suggestion
namespace {

std::string GetFloatArg()
{
    const auto& tmp = env::value(MIOPEN_TEST_FLOAT_ARG);
    if(tmp.empty())
    {
        return "";
    }
    return tmp;
}

} // namespace

using GPU_GLU_bwd_FP32 = GLUBwdTest<float>;
using GPU_GLU_bwd_FP16 = GLUBwdTest<float16>;
using GPU_GLU_bwd_BFP16 = GLUBwdTest<bfloat16>;
```

This will solve the `duplicate symbol` problem

Reply:
The same changes should be made to the `glu_fwd.cpp`

Reply:
I have updated based on your comments and merged `glu_fwd` and `glu_bwd` into 1 file